onprc_ehr/src/org/labkey/onprc_ehr/notification/ColonyAlertsNotification.java

```diff
@@ -1061,7 +1099,7 @@ protected void pmicSchedulerAlert(final Container c, User u, final StringBuilder
             return;
         }
         //Daily events query
-        TableInfo ti = QueryService.get().getUserSchema(u, c, "extscheduler").getTable("PMIC_Scheduler");
+        TableInfo ti = QueryService.get().getUserSchema(u, c, "extscheduler").getTable("PMIC_Scheduler_Daily");
```

Comment:
I didn't see PMIC_Scheduler_Daily created anywhere in this merge, & further investigating looks like it is not there in the module either, so perhaps stored in the db. Makes sense for it to be part of the module, unless its outside the scope of this merge.